File: c++/rust-examples-using-c++/tst/chapter1/hello_world-test.cpp
#include "gtest/gtest.h"
#include "chapter1/hello_world.h"

TEST(HelloWorldTest, test1){
     EXPECT_EQ(HelloWorld::shout(), "Hello World!");
}
TEST(HelloWorldTest, test2){
     EXPECT_NE(HelloWorld::shout(), "Hello World");
}

File: java/rustExamplesUsingJava/src/test/java/com/sm/rust/examples/using/java/chapter1/HelloWorldTest.java
package com.sm.rust.examples.using.java.chapter1;

import static org.junit.Assert.assertEquals;

import org.junit.jupiter.api.Test;

import com.sm.rust.examples.using.java.chapter1.HelloWorld;

public class HelloWorldTest {
	
    private HelloWorld helloWorld;
    HelloWorldTest(){
    	this.helloWorld = new HelloWorld();
    }
    
	@Test
	public void helloWorldTest() {
		assertEquals("Hello World!", helloWorld.helloWorld());
	}
}
